server/app.js

```diff
@@ -1,16 +1,21 @@
+require('dotenv').config();
```

Comment:
this actually is run in www/bin - so it should already be happening. Was the bug happening before only in circleci or also locally?

Reply:
It happened when running npm test locally as well. 

My theory is that since the connectDB() function is run in app.js prior to www/bin, dotenv is not configured prior to making use of the variable. We can see in the log Ryan's console.log outputs before the listening statement. Do you think that's what happened?

Reply:
I tried fiddling around with some things, but could not get the tests to pass without including that at the top of app.js